I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/d1s1.rpy", line 149, in script
    call screen deduction_game_menu
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "renpy/common/00action_other.rpy", line 585, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/deduction_game.rpy", line 129, in check_solution
    if sentence[key] != solution[key]:
KeyError: 'zone 2'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/d1s1.rpy", line 149, in script
    call screen deduction_game_menu
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 2259, in execute
    self.call("execute")
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 2241, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\statements.py", line 342, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\exports.py", line 3347, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\display\core.py", line 3582, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\display\core.py", line 4543, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\display\layout.py", line 1179, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\display\layout.py", line 1179, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\display\layout.py", line 1179, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\display\screen.py", line 770, in event
    rv = self.child.event(ev, x, y, st)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\display\layout.py", line 1179, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\display\behavior.py", line 1142, in event
    return handle_click(self.clicked)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\display\behavior.py", line 1075, in handle_click
    rv = run(action)
  File "C:\Users\blue_\Downloads\renpy-8.1.3-sdk\renpy\display\behavior.py", line 379, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_other.rpy", line 585, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/deduction_game.rpy", line 129, in check_solution
    if sentence[key] != solution[key]:
KeyError: 'zone 2'

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
Rounded GUI 1.1.0
Fri Nov 24 15:23:35 2023
